store/postgres/src/chain_head_listener.rs

```diff
@@ -89,7 +89,7 @@ impl ChainHeadUpdateListener {
                         let logger = logger.clone();
                         let subscribers = subscribers.clone();
 
-                        sender.send(update.clone()).then(move |result| {
+                        tokio::spawn(sender.send(update.clone()).then(move |result| {
```

Comment:
An alternative approach could be `futures::unordered` maybe? But even that could hold progress back if one of the futures blocks. So no, this is probably better.

Reply:
One concern I would have is that we spawn more and more tasks here over time, and still eat up more and more memory until it's too much.